src/collar/oc_particle.lsl

```diff
@@ -428,6 +448,26 @@ DebugOutput(key kID, list ITEMS){
     llInstantMessage(kID, llGetScriptName() +final);
 }
 
+HandleOccCMD(string sCMD){
```

Comment:
How re-usable is this by non-cuff attachments?  We get a lot of requests for interacting with other things like piercings.  I wouldn't want to add a function like this for every such item.

Reply:
in oc_particle there is only the chain handling added. It's like lockguard or lockmeister but it can simply create chains between attachments. It's needed in the collar as well so the attachments can also create chains to the collar. So it can also be used for piercings to create chains :-)
There is no need to add another function. This pr was only to seperate chain textures, because the default silk-ribbon looks pretty bad on short distances.